File: javascript/javascript-find-sum-odd-below-number/example3.js
// https://codevscolor.com/javascript-find-sum-odd-below-number
function findSum(no) {
  let sum = 0;

  for (var i = 1; i < no; i += 2) {
    sum += i;
  }
  return sum;
}

console.log(findSum(100));

File: java/java-find-sum-fibonacci-series/Example3.java
// https://codevscolor.com/java-find-sum-fibonacci-series

import java.util.Scanner;

class Example3 {

    private static int findSum(int currentValue, int prevValue, int i, int n) {
        if (n <= 0)
            return 0;
        if (n == 1)
            return 1;

        if (i == n)
            return 0;

        return currentValue + findSum(currentValue + prevValue, currentValue, i + 1, n);
    }

    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter the value of n: ");
            int n = sc.nextInt();

            System.out.println("Sum = " + findSum(1, 0, 1, n));
        }

    }
}